reporter/otlp_reporter.go

```diff
@@ -732,6 +742,16 @@ func getTraceLabels(stringMap map[string]uint32, i traceInfo) []*pprofextended.L
 		})
 	}
 
+	if i.pid != 0 {
+		pidIdx := getStringMapIndex(stringMap, "pid")
+		pidValueIdx := getStringMapIndex(stringMap, string(i.pid))
+
+		labels = append(labels, &pprofextended.Label{
```

Comment:
I think this is indeed a label that would make a lot of sense to add, including upstream :+1:

Reply:
It's weird to add the label to `OTLPReporter` but not `DatadogReporter`, is it intended ?
For pid, label key should be `process_id` to be consistent with [ddprof](https://github.com/DataDog/ddprof/blob/main/src/pprof/ddprof_pprof.cc#L241).